mne/epochs.py

```diff
@@ -779,10 +779,19 @@ class Epochs(_BaseEpochs, ToDataFrameMixin):
         Return Epochs object with a subset of epochs corresponding to an
         experimental condition as specified by 'name'.
 
+        If conditions are hierarchical (separated by '/', e.g. 'audio/left',
```

Comment:
I wouldn't explicitly say hierarchical, they don't need to be in a hierarchy -- a user can choose whether the `/` implies hierarchy or just condition tagging if they wish, the code works the same either way. So I think if you just say `"If conditions have names containing '/', ...`